onmt/translate/translation.py

```diff
@@ -54,10 +54,11 @@ def from_batch(self, translation_batch):
                len(translation_batch["predictions"]))
         batch_size = batch.batch_size
 
-        preds, pred_score, attn, gold_score, indices = list(zip(
+        preds, pred_score, attn, copy, gold_score, indices = list(zip(
```

Comment:
Think we need a better solution here. Also what happens if copy is not used, isn't the zip truncated?

Reply:
When copy is not used, the `copy` variable will be the list like [[], [], [], ......, [], []], the length of which is batch_size. Therefore it will not affect other parts.

Now, the way of introducing `copy_debug` is the same with `attn_debug`. So we make several changes in translation.py. I am trying my best but still don't figure out more elegant ways.